templates/3_Pixie_Cluster_Cells.ipynb

```diff
@@ -854,7 +854,7 @@
     "        xr_channel_names=None,\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        plot_utils.plot_pixel_cell_cluster(
</pre><p>You should check with Candace if she wants this to default to erode=True or False. I think the cell masks look better eroded. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1052/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
Got Candace's approval for cell mask erosion in the Pixie Pipeline notebooks.